aten/src/ATen/native/cuda/UpSample.cuh

```diff
@@ -94,11 +94,16 @@ __host__ __forceinline__ static accscalar_t area_pixel_compute_scale(
     int output_size,
     bool align_corners,
     const c10::optional<double> scale) {
-  if (output_size > 1) {
-    return align_corners ? (accscalar_t)(input_size - 1) / (output_size - 1)
-                         :  compute_scales_value<accscalar_t>(scale, input_size, output_size);
-  } else {
-    return static_cast<accscalar_t>(0);
+  if(align_corners){
+    if(output_size > 1){
+      return (accscalar_t)(input_size - 1) / (output_size - 1);
+    }
+    else{
+      return static_cast<accscalar_t>(0);
+    }
+  }
+  else{
+    return compute_scales_value<accscalar_t>(scale, input_size, output_size);
```

Comment:
Same here: please keep brace formatting consistent with elsewhere (spaces before curly braces and `} else {`)

Reply:
ok, I have added a test sample, can you see if there is any problem？

Reply:
Test looks okay to me